tensorflow/compiler/xla/service/gpu/tests/gpu_fusion_test.cc

```diff
@@ -15,6 +15,8 @@ limitations under the License.
 
 #include <utility>
 
+#include "tensorflow/compiler/xla/service/gpu/gpu_fusible.h"
```

Comment:
It looks to me that there needs to be a corresponding BUILD file change for these two header includes?

Reply:
I expected that it would ask for it, but it was building without asking for it.
I added them.